resources/views/index.blade.php

```diff
@@ -53,7 +53,7 @@
                   <li class="nav-item">
```

Comment:
Might be good to add a nav link for organizers as well as mentors, optional though.

Reply:
![image](https://user-images.githubusercontent.com/22565214/42740118-7dcf2bc2-8857-11e8-972a-9412866e39f5.png)
Added nav-bar link to organizers